Crash when entering into subscriptions

### Steps to reproduce

1. Latest debug
2. Try opening Subscription tab

### Expected behavior

Should not crash

### Actual behavior

Crashed with

`androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment com.github.libretube.ui.fragments.SubscriptionsFragment: calling Fragment constructor caused an exception
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:681)
	at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:57)
	at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:507)
	at androidx.navigation.fragment.FragmentNavigator.createFragmentTransaction(FragmentNavigator.kt:250)
	at androidx.navigation.fragment.FragmentNavigator.navigate(FragmentNavigator.kt:185)
	at androidx.navigation.fragment.FragmentNavigator.navigate(FragmentNavigator.kt:164)
	at androidx.navigation.NavController.navigateInternal(NavController.kt:260)
	at androidx.navigation.NavController.navigate(NavController.kt:1719)
	at androidx.navigation.NavController.navigate(NavController.kt:1545)
	at androidx.navigation.NavController.navigate(NavController.kt:1472)
	at androidx.navigation.NavController.navigate(NavController.kt:1454)
	at androidx.navigation.NavController.navigate(NavController.kt:1437)
	at com.github.libretube.ui.activities.MainActivity.navigateToBottomSelectedItem(MainActivity.kt:482)
	at com.github.libretube.ui.activities.MainActivity.onCreate$lambda$1(MainActivity.kt:131)
	at com.github.libretube.ui.activities.MainActivity.$r8$lambda$sSW0nh76RJJnmyJ72uoyVjS5svM(Unknown Source:0)
	at com.github.libretube.ui.activities.MainActivity$$ExternalSyntheticLambda2.onNavigationItemSelected(Unknown Source:2)
	at com.google.android.material.navigation.NavigationBarView$1.onMenuItemSelected(NavigationBarView.java:291)
	at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:836)
	at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:159)
	at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:987)
	at com.google.android.material.navigation.NavigationBarMenuView$1.onClick(NavigationBarMenuView.java:139)
	at android.view.View.performClick(View.java:7498)
	at android.view.View.performClickInternal(View.java:7471)
	at android.view.View.access$3700(View.java:843)
	at android.view.View$PerformClick.run(View.java:29098)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:233)
	at android.os.Looper.loop(Looper.java:344)
	at android.app.ActivityThread.main(ActivityThread.java:8248)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:589)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1071)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:663)
	... 32 more
Caused by: java.lang.ClassCastException: java.lang.Long cannot be cast to java.lang.Integer
	at android.app.SharedPreferencesImpl.getInt(SharedPreferencesImpl.java:321)
	at com.github.libretube.helpers.PreferenceHelper.getInt(PreferenceHelper.kt:67)
	at com.github.libretube.ui.fragments.SubscriptionsFragment.<init>(SubscriptionsFragment.kt:38)
	... 35 more `

### LibreTube version

Latest debug 

### Android version

Android 12

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/constants/PreferenceKeys.kt
+++ b/app/src/main/java/com/github/libretube/constants/PreferenceKeys.kt
@@ -116,7 +116,7 @@ object PreferenceKeys {
     const val LAST_STREAM_VIDEO_ID = "last_stream_video_id"
     const val LAST_WATCHED_FEED_TIME = "last_watched_feed_time"
     const val HIDE_WATCHED_FROM_FEED = "hide_watched_from_feed"
-    const val SELECTED_FEED_FILTER = "selected_filer_feed"
+    const val SELECTED_FEED_FILTER = "filer_feed"
     const val FEED_SORT_ORDER = "sort_oder_feed"
 
     /**

